src/pharmpy/model/external/nonmem/table.py

```diff
@@ -209,7 +209,7 @@ class PhiTable(NONMEMTable):
     @property
     def iofv(self):
         df = self._df.copy(deep=True)
-        # Remove all zero lines. These are from individuals with no observations
+        # Remove all zero lines. These are from individuals with no observations.
```

Comment:
Touching a line of code change `git blame` obscuring the original author of the comment. So for little added value I would like to keep this without changes